packages/expo-router/src/global-state/routing.ts

```diff
@@ -76,7 +77,21 @@ export function setParams(this: RouterStore, params: Record<string, string | num
   return (this.navigationRef?.current?.setParams as any)(params);
 }
 
-export function linkTo(this: RouterStore, href: string, event?: string) {
+export type LinkToOptions = {
+  event?: string;
+
+  /**
+   * Relative URL references are either relative to the directory or the document. By default, relative paths are relative to the document.
+   * @see: https://developer.mozilla.org/en-US/docs/Web/API/URL_API/Resolving_relative_references
```

Comment:
```suggestion
   * @see: [MDN's documentation on Resolving relative references to a URL](https://developer.mozilla.org/en-US/docs/Web/API/URL_API/Resolving_relative_references).
```

Let's add the text to support the link here.